py/picca/delta_extraction/expected_fluxes/true_continuum.py

```diff
@@ -152,13 +152,30 @@ def compute_expected_flux(self, forests):
         -----
         ExpectedFluxError if Forest.wave_solution is not 'lin' or 'log'
         """
-        context = multiprocessing.get_context('fork')
-        pool = context.Pool(processes=self.num_processors)
+        in_nside = 16
+        healpixes = np.array([healpy.ang2pix(in_nside,
```

Comment:
This is also computed in function `read_true_continuum` so maybe we don't need to compute it twice. How about we compute it only here and pass the healpix as an argument there?